### ISSUE TITLE
`0.2.0` and `0.1.20` worked, but becomes broken

### ISSUE THREAD

**Issue by AlexWayfer** [codecov/codecov-ruby#72]
https://github.com/AlexWayfer/alt_memery/pull/9

https://cirrus-ci.com/task/6006359470637056?command=test#L0

![image](https://user-images.githubusercontent.com/6510020/87314228-99a1ba00-c52b-11ea-9bb3-2cd467bff377.png)

**Comment by AlexWayfer** [comment]
`0.1.20` is affected too:

https://github.com/AlexWayfer/module_methods/pull/5

https://cirrus-ci.com/task/6445327207104512

![image](https://user-images.githubusercontent.com/6510020/87314438-de2d5580-c52b-11ea-9ba0-861547d60723.png)

**Comment by ShockwaveNN** [comment]
Same here, seems there is some problems on back-end

**Comment by steveburkett** [comment]
seeing this problem right now. 

```
->  Uploading to Codecov
https://codecov.io//upload/v2?token=secret&flags&package=ruby-0.2.0&branch=sb%2F16-setup-ci&commit=c987544b38779b9bc799beecb09544dfca381126
/Users/steveburkett/.rvm/gems/ruby-2.6.5@event-capture-api/gems/json-2.3.1/lib/json/common.rb:263:in `parse': 784: unexpected token at '<!DOCTYPE html> (JSON::ParserError)
<html lang="en">
<head>
```

**Comment by thomasrockhu** [comment]
Hi @steveburkett, can you share a build URL for the above?

**Comment by thomasrockhu** [comment]
@AlexWayfer, for your case, the token you are providing is not a UUID. Can you double-check how you are supplying `CODECOV_TOKEN`?

**Comment by AlexWayfer** [comment]
> @AlexWayfer, for your case, the token you are providing is not a UUID. Can you double-check how you are supplying `CODECOV_TOKEN`?

Yes, it's [encrypted by Cirrus CI](https://cirrus-ci.org/guide/writing-tasks/#encrypted-variables).

**Comment by thomasrockhu** [comment]
We just released `0.2.1` of the gem, would you be able to see what happens now?

**Comment by steveburkett** [comment]
@thomasrockhu that fixed it! awesome!

**Comment by AlexWayfer** [comment]
> We just released `0.2.1` of the gem, would you be able to see what happens now?

Now it works as I see, thank you.
